pkg/api/convertertoapi.go

```diff
@@ -776,15 +780,8 @@ func convertV20170131CustomProfile(v20170131 *v20170131.CustomProfile, api *Cust
 	api.Orchestrator = v20170131.Orchestrator
 }
 
-func convertV20170701CertificateProfile(v20170701 *v20170701.CertificateProfile, api *CertificateProfile) {
-	api.CaCertificate = v20170701.CaCertificate
-	api.APIServerCertificate = v20170701.APIServerCertificate
-	api.APIServerPrivateKey = v20170701.APIServerPrivateKey
-	api.ClientCertificate = v20170701.ClientCertificate
```

Comment:
Just chatted with @JackQuincy and we will leave this out for now to be consistent with 20170131